src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -395,16 +391,16 @@ trait GraphDAO {
             case Some(nonEntityVertex) => throw new RawlsException(s"property [${paramName}] of class [${classT.fullName}] does not reference non entity vertex ${nonEntityVertex}")
           }
 
-        case workflowSymbol if workflowSymbol =:= typeOf[Seq[Workflow]] => deserializeDomainObjects[Workflow](vertex, workflowEdge, workspaceName)
-        case workflowFailureSymbol if workflowFailureSymbol =:= typeOf[Seq[WorkflowFailure]] => deserializeDomainObjects[WorkflowFailure](vertex, workflowFailureEdge, workspaceName)
-        case msConfigSymbol if msConfigSymbol =:= typeOf[MethodRepoConfiguration] => tryLoad(deserializeDomainObjects[MethodRepoConfiguration](vertex, methodRepoConfigEdge, workspaceName).headOption)
-        case msMethodSymbol if msMethodSymbol =:= typeOf[MethodRepoMethod] => tryLoad(deserializeDomainObjects[MethodRepoMethod](vertex, methodRepoMethodEdge, workspaceName).headOption)
+        case workflowSymbol if workflowSymbol =:= typeOf[Seq[Workflow]] => deserializeDomainObjects[Workflow](vertex, workflowEdge)
+        case workflowFailureSymbol if workflowFailureSymbol =:= typeOf[Seq[WorkflowFailure]] => deserializeDomainObjects[WorkflowFailure](vertex, workflowFailureEdge)
+        case msConfigSymbol if msConfigSymbol =:= typeOf[MethodRepoConfiguration] => tryLoad(deserializeDomainObjects[MethodRepoConfiguration](vertex, methodRepoConfigEdge).headOption)
+        case msMethodSymbol if msMethodSymbol =:= typeOf[MethodRepoMethod] => tryLoad(deserializeDomainObjects[MethodRepoMethod](vertex, methodRepoMethodEdge).headOption)
 
         case attributeSeq if attributeSeq <:< typeOf[Seq[Attribute]] => vertex.getVertices(Direction.OUT, paramName).headOption.map(deserializeSeq).getOrElse(Seq.empty)
 
         case optionSymbol if optionSymbol <:< typeOf[Option[_]] => vertexProperty
 
-        case workspaceNameSymbol if workspaceNameSymbol =:= typeOf[WorkspaceName] => workspaceName.getOrElse(throw new RawlsException("workspace name not supplied but is required"))
+        case workspaceNameSymbol if workspaceNameSymbol =:= typeOf[WorkspaceName] => throw new RawlsException(s"workspace name not supplied but is required for ${classT.fullName}.${paramName}")
```

Comment:
If this is the case then does it make sense to say the name is "not supplied but is required"? Because it sounds to me that if the exception is thrown, the opposite is the case.